Source/WebCore/dom/Element.cpp

```diff
@@ -2093,7 +2093,7 @@ void Element::attributeChanged(const QualifiedName& name, const AtomString& oldV
             }
         } else if (name == HTMLNames::partAttr)
             partAttributeChanged(newValue);
-        else if (document().settings().ariaReflectionForElementReferencesEnabled() && (isElementReflectionAttribute(name) || isElementsArrayReflectionAttribute(name))) {
```

Comment:
Why is it OK to drop this check? The flag still exists and can be toggled from the experimental features menu AFAIK.

Reply:
The popover attribute uses a different flag, I could either add both flags to the check, or remove the check entirely.

Reply:
You cannot remove the flag check for an aria feature like that, without removing the feature flag entirely (which you didn't). And I think it may be too early to drop the aria feature flag since the feature is super recent.